src/actions/index.js

```diff
@@ -0,0 +1,9 @@
+import {ACTIONS as STATE_MANAGEMENT_ACTIONS, stateLoad} from './stateManagement';
+
+export {
+  STATE_MANAGEMENT_ACTIONS,
+};
+
+export {
+  stateLoad,
+};
```

Comment:
Give the components in their separate `components` domain access to the action type keys.